RELEASE.md

```diff
@@ -14,8 +14,9 @@ Example `version`
 
 - Update and review `CHANGELOG.md`:
 
-      gem install github_changelog_generator
-      github_changelog_generator -u scientific-python -p spin --since-tag=<last tag>
+      changelist scientific-python/spin  <v0.0> main --version <0.1> >> CHANGELOG.md
+
+  where <v0.0> is the last release and <0.1> is the new one.
```

Comment:
```suggestion
  replacing v0.0 with the previous release and v0.1 is the new one.
```

Not sure if that should be `v0.1` or `0.1`?

Reply:
The `0.1` is used in the CHANGELOG and the `v0.0` is a tag. We've been prepending `v` to the release tags, which is pretty common and we should stick with it. I think the release number shouldn't have the `v` prepended. If it does, then the CHANGELOG will say things like:
```
We're happy to announce the release of spin v0.1!
```

I think it looks better the way I had it
```
We're happy to announce the release of spin 0.1!
```

Feel free to make a PR to make whatever changes you want. I don't have a strong opinion on this. I will copy your changes to the other sites to keep things consistent.